source/_components/media_player.epson.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Epson"
+description: "Instructions on how to integrate Epson projector into Home Assistant."
+date: 2018-03-24 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: epson.png
+ha_category: Media Player
+featured: true
+ha_release: "0.66"
```

Comment:
This feature won't make it into 0.66 since it has been frozen already.
Maybe update it to 0.67?